package.json

```diff
@@ -4,11 +4,16 @@
   "description": "📅 Calendar link generator for popular services, like Google, Outlook, and Yahoo.",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "source": "src/index.ts",
+  "exports": "dist/index.modern.js",
+  "umd:main": "dist/index.umd.js",
   "files": [
     "dist"
   ],
   "scripts": {
-    "build": "tsc",
+    "build:umd": "microbundle --external none --format modern,umd",
+    "build:cjs": "tsc -m commonjs",
+    "build": "run-p build:*",
```

Comment:
Alternatively: `npm run build:umd && npm run build:cjs` but it'll take a bit more time.